metrics.py

```diff
@@ -0,0 +1,56 @@
+import time
```

Comment:
Is this file meant to be part of this PR?

Reply:
No, it's not. Thanks! Idk, how it slipped in there...